charts/flyte-core/templates/propeller/webhook.yaml

```diff
@@ -103,6 +103,9 @@ spec:
             allowPrivilegeEscalation: false
             capabilities:
               drop: ["ALL"]
+          {{- if .Values.webhook.resources -}}
```

Comment:
nit: Not important in this case but we could also do the following. 

```yaml
{{- with .Values.webhook.resources }}
resources: {{- toYaml . | nindent 12 }}
{{- end }}
```

This is not really that important as it's just syntactic difference, only really benefits for more complex config. We can always update later.

Reply:
thanks @mhotan updated to address both your comments, although having a hard time reasoning about what sensible defaults would be here esp given the recent ephemeral storage incident. figure it might be safer to err on overprovisioning for now